docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -136,109 +181,427 @@ These variables apply only when using NCO mode (i.e. when ``RUN_ENVIR`` is set t
+   Decorrelation spatial scales in meters. 
+
+``SPPT_TSCALE``: (Default: "21600") 
+   Decorrelation timescale in seconds. Corresponds to the variable ``sppt_tau`` in ``input.nml``.
+   
+``SPPT_INT``: (Default: "3600") 
+   Interval in seconds to update random pattern (optional parameter). Perturbations still get applied at every time-step. Corresponds to the variable ``spptint`` in ``input.nml``.
+
+``SPPT_SFCLIMIT``: (Default: "TRUE")
+   When "TRUE", tapers the SPPT perturbations to zero at the model’s lowest level, which reduces model crashes. 
+
+``USE_ZMTNBLCK``: (Default: "FALSE")
+   When "TRUE", do not apply perturbations below the dividing streamline that is diagnosed by the gravity wave drag, mountain blocking scheme
+
+Stochastic Kinetic Energy Backscatter (SKEB) Parameters
+----------------------------------------------------------
+
+``DO_SKEB``: (Default: "FALSE")
+   Flag to turn Stochastic Kinetic Energy Backscatter (SKEB) on or off. SKEB adds wind perturbations to the model state. Perturbations are random in space/time, but amplitude is determined by a smoothed dissipation estimate provided by the :term:`dynamical core`. SKEB addresses errors in the dynamics more active in the mid-latitudes.
+
+``ISEED_SKEB``: (Default: "3")
+   Seed for setting the SHUM random number sequence.
+
+``SKEB_MAG``: (Default: "0.5") 
+   Amplitude of random patterns. Corresponds to the variable ``skeb`` in ``input.nml``.
+
+``SKEB_LSCALE``: (Default: "150000")
+   Decorrelation spatial scale in meters. 
+
+``SKEB_TSCALE``: (Default: "21600")
+   Decorrelation timescale in seconds. Corresponds to the variable ``skeb_tau`` in ``input.nml``.
+
+``SKEB_INT``: (Default: "3600")
+   Interval in seconds to update random pattern (optional). Perturbations still get applied every time-step. Corresponds to the variable ``skebint`` in ``input.nml``.
+
+``SKEBNORM``: (Default: "1")
+   Patterns:
+      * 0-random pattern is stream function
+      * 1-pattern is K.E. norm
+      * 2-pattern is vorticity
+
+``SKEB_VDOF``: (Default: "10")
+   The number of degrees of freedom in the vertical for the SKEB random pattern. 
+
+..
+   COMMENT: The vertical what?
+
+.. _SPP:
+
+SPP Stochastic Physics Parameterizations
+--------------------------------------------
+
+Set default Stochastically Perturbed Parameterizations (SPP) stochastic physics options. Unlike :ref:`SPPT physics <SPPT>`, SPP is applied as part of the physics run, not post. SPP perturbs specific tuning parameters within a physics scheme (unlike `SPPT <SPPT>`, which multiplies overall physics tendencies by a random numer *after* the call to the physics suite). Each SPP option is an array, applicable (in order) to the scheme/parameter listed in ``SPP_VAR_LIST``. Enter each value of the array in ``config.sh`` as shown below without commas or single quotes (e.g., ``SPP_VAR_LIST=( "pbl" "sfc" "mp" "rad" "gwd"`` ). Both commas and single quotes will be added by Jinja when creating the namelist.
```

Comment:
Change: "Unlike :ref:`SPPT physics <SPPT>`, SPP is applied as part of the physics run, not post. SPP perturbs specific tuning parameters within a physics scheme (unlike `SPPT <SPPT>`,"

To: "Unlike :ref:`SPPT physics <SPPT>`, SPP is applied within the physics, not afterward. SPP perturbs specific tuning parameters within a physics parameterization(unlike `SPPT <SPPT>`,"

which multiplies overall physics tendencies by a random numer *after* the call to the physics suite) -> which multiplies overall physics tendencies by a random perturbation field *after* the call to the physics suite)

Change: Each SPP option is an array, applicable (in order) to the scheme/parameter listed in ``SPP_VAR_LIST``

To: Each SPP option is an array, applicable (in order) to the RAP/HRRR-based parameterization listed in ``SPP_VAR_LIST``

Reply:
Done! Just to clarify, "pbl" "sfc" "mp" "rad" and "gwd" are perturbation fields, right?

Reply:
Those are the parameterizations that will be perturbed.  Within each parameterization, there are multiple parameters or variables that will be perturbed, but that information will go in the ccpp-physics documentation.